app/grandchallenge/reader_studies/views.py

```diff
@@ -398,6 +398,27 @@ def generate_hanging_list(self, request, pk=None):
         )
         return Response({"status": "Hanging list generated."},)
 
+    @action(detail=True, methods=["patch"])
+    def remove_image(self, request, pk=None):
+        image_id = request.data.get("image")
+        reader_study = self.get_object()
+        try:
+            reader_study.images.remove(Image.objects.get(id=image_id))
+            messages.add_message(
+                request, messages.SUCCESS, "Image removed from reader study."
+            )
+            return Response({"status": "Image removed from reader study."},)
+        except Image.DoesNotExist:
+            pass
```

Comment:
It's best not to do try-except-pass. Replace `pass` with the fall through block (L413-420) and return in the except block.